packages/core/src/lib/passkey/browser-script.js

```diff
@@ -0,0 +1,225 @@
+//@ts-check
```

Comment:
I am still thinking about an easier solution for customized sign in page to have this feature. I think most of our developers are using custom pages. 🤔 maybe we can introduce `@auth/core/browser` and export this script & the SimpleWebauthnBrowser script? cc @balazsorban44